src/Ormolu/Printer/Meat/Declaration/Value.hs

```diff
@@ -858,21 +858,25 @@ p_hsExpr' s = \case
 
 p_patSynBind :: PatSynBind GhcPs GhcPs -> R ()
 p_patSynBind PSB {..} = do
-  let rhs = do
+  let rhs conSpans = do
         space
+        let pattern_def_spans = getLocA psb_def : conSpans
```

Comment:
```suggestion
        let pattern_def_spans = [getLocA psb_id, getLocA psb_def] <> conSpans
```

The snippet
```haskell
{-# LANGUAGE PatternSynonyms #-}

pattern NumbersNumbersEverywhereeeeeeeeeeeeeeeee =
  B 1234 1243 1234 1234 1234 1234 1234 1234 1234 1234
```
a fixed point, but now it is formatted to
```haskell
{-# LANGUAGE PatternSynonyms #-}

pattern NumbersNumbersEverywhereeeeeeeeeeeeeeeee = B 1234 1243 1234 1234 1234 1234 1234 1234 1234 1234
```
If the LHS has arguments, the issue disappears. I added a quick fix which I think fixes the issue (it also requires one update in the test suite).